application/freshdesk/v0/config/tasks.json

```
@@ -0,0 +1,2455 @@
+      "instillEditOnNodeFields": ["ticket-id"],
+      "properties": {
+        "ticket-id": {
+          "$ref": "#/$defs/ticket/ticket-id",
+          "instillAcceptFormats": ["integer"],
+          "instillUpstreamTypes": ["value", "reference"],
+          "instillUIMultiline": false,
+          "instillUIOrder": 0
+        }
+      },
+      "required": ["ticket-id"],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Ticket information",
+      "instillUIOrder": 0,
+      "properties": {
+        "subject": {
+          "$ref": "#/$defs/ticket/subject",
+          "instillFormat": "string",
+          "instillUIOrder": 0
+        },
+        "description-text": {
+          "$ref": "#/$defs/ticket/description-text",
+          "instillFormat": "string",
+          "instillUIOrder": 1
+        },
+        "source": {
+          "$ref": "#/$defs/ticket/source",
+          "instillFormat": "string",
+          "instillUIOrder": 2
+        },
+        "status": {
+          "$ref": "#/$defs/ticket/status",
+          "instillFormat": "string",
+          "instillUIOrder": 3
+        },
+        "priority": {
+          "$ref": "#/$defs/ticket/priority",
+          "instillFormat": "string",
+          "instillUIOrder": 4
+        },
+        "ticket-type": {
+          "$ref": "#/$defs/ticket/ticket-type",
+          "instillFormat": "string",
+          "instillUIOrder": 5
+        },
+        "association-type": {
+          "$ref": "#/$defs/ticket/association-type",
+          "instillFormat": "string",
+          "instillUIOrder": 6
+        },
+        "associated-ticket-list": {
+          "$ref": "#/$defs/ticket/associated-ticket-list",
+          "instillFormat": "array:integer",
+          "items": {
+            "title": "Associated Ticket",
+            "type": "integer",
+            "description": "The ID of the ticket associated with the current ticket",
+            "instillFormat": "integer"
+          },
+          "instillUIOrder": 7
+        },
+        "tags": {
+          "$ref": "#/$defs/ticket/tags",
+          "instillFormat": "array:string",
+          "items": {
+            "title": "Tag",
+            "type": "string",
+            "description": "Tag associated with the ticket",
+            "instillFormat": "string"
+          },
+          "instillUIOrder": 8
+        },
+        "cc-emails": {
+          "$ref": "#/$defs/ticket/cc-emails",
+          "instillFormat": "array:string",
+          "items": {
+            "title": "CC Email",
+            "type": "string",
+            "description": "One of the email address added in the 'cc' field of the incoming ticket email",
+            "instillFormat": "string"
+          },
+          "instillUIOrder": 9
+        },
+        "forward-emails": {
+          "$ref": "#/$defs/ticket/forward-emails",
+          "instillFormat": "array:string",
+          "items": {
+            "title": "Foward Email",
```

```suggestion
            "title": "Forward Email",
```
Though it seems not to show in frontend.